src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -89,14 +87,14 @@ export default function Autocomplete({
       disabled={disabled}
       id={id}
       options={values}
-      getOptionLabel={(option: any) => (option ? (option.label || option) : '')}
+      getOptionLabel={(option: any) => (option ? option.label || option : '')}
       onChange={onChangeHandler}
       onInputChange={(event: any, value: any) => freeSolo && onChangeHandler(event, value)}
-      inputValue={(selected as Option)?.label || selected}
+      value={(selected as Option)?.label || selected}
```

Comment:
Interesting thanks for fixing this.
The selected value highlight (in storybook) is not showing now - probably something I missed.
Can you add this one line 
```
  isEqual: (a: any, b: any) => a.value === b.value,
```
to Complex.args? https://github.com/terraware/web-components/blob/main/src/stories/Autocomplete.stories.tsx#L49

Reply:
Oh, yes, let me try that